sea-orm-cli/Cargo.toml

```diff
@@ -34,7 +34,7 @@ path = "src/bin/sea.rs"
 required-features = ["cli", "codegen"]
 
 [dependencies]
-clap = { version = "3.2", default-features = false, features = ["std", "env", "derive"], optional = true }
+clap = { version = "4.1.4", features = ["env", "derive", "deprecated"], optional = true }
```

Comment:
What `deprecated` feature we used? If any, can we refactor the implementation of it so that we does not need the `deprecated` feature being enabled?

Reply:
Clap deprecated feature is just there to warn about the things you are using which are deprecated. I was using it just to get warning about using a clapv3 feature. I removed it from the PR. 
https://docs.rs/clap/latest/clap/_features/